### DISCUSSION TITLE
请问为什么或有这种emoji显示不出来的情况呀，怎么解决哇

### DISCUSSION THREAD

**Discussion by Jamjoker** [fcitx5-android/fcitx5-android#561] [Q&A | 问答] [answered]
![Screenshot_2024-08-10-23-53-31-56_e39d2c7de19156b0683cd93e8735f348.jpg](https://github.com/user-attachments/assets/cdaa7cf7-f666-4faa-a408-1e4d19e77246)

**Comment by rocka** [comment, marked as answer]
因为系统字体不支持新版的 emoji ，参考 https://github.com/fcitx5-android/fcitx5-android/issues/185
